src/index.js

```diff
@@ -215,13 +222,17 @@ class Explosion extends React.PureComponent<Props, State> {
           });
           const containerTransform = [{translateX: left}, {translateY: top}];
           const transform = [{rotateX}, {rotateY}, {rotate: rotateZ}, {translateX}];
+          if (Platform.OS === 'android') {
+            transform.push({ perspective: 100 });
```

Comment:
can you explain what this new transform property does please ? I'm not sure it's related to the bugfix you did, but more about something (nice) related to UI deep perception of confettis, isn't it ?